internal/usecase/client/ui_input.go

```
@@ -0,0 +1,163 @@
+	case "Tab":
+		return "\t", true
+	case "Backspace":
+		return "\x7f", true
+	case "Space":
+		return " ", true
+	case "PageUp":
+		return "\x1b[5~", true
+	case "PageDown":
+		return "\x1b[6~", true
+	}
+	final := ""
```

golangci-lint (2.13.2)

[error] assigned to final, but reassigned without using the value

(wastedassign)